packages/clerk-js/src/ui/elements/InputWithIcon.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import { Flex, Input } from '../customizables';
+import { Box, Flex, Input } from '../customizables';
 import type { PropsOfComponent } from '../styledSystem';
 
 type InputWithIcon = PropsOfComponent<typeof Input> & { leftIcon?: React.ReactElement };
```

Comment:
Comment from @alexcarpenter -> we might as well add some descriptors for this element

There's already a default one for `Input`, so I presume it makes sense to forward a custom descriptor here so we can target it as "search" for `OrganizationProfile`